src/org/traccar/model/CalendarPermission.java

```diff
@@ -16,17 +16,7 @@
  */
 package org.traccar.model;
 
-public class CalendarPermission {
-
-    private long userId;
-
-    public long getUserId() {
-        return userId;
-    }
-
-    public void setUserId(long userId) {
-        this.userId = userId;
-    }
+public class CalendarPermission extends BaseUserPermission {
```

Comment:
Another thought... can we get rid of permission classes? Can we just use Map with keys-value pairs? I believe it will be serialized into same JSON object?